### ISSUE TITLE
Memory leak / endless loop

### ISSUE THREAD

**Issue by agoldis** [kenperkins/winston-papertrail#65]
Using `util.inspect` without depth limit can cause memory leak / endless flooding in case an object has circular dependency 

lines `335`, `331` and `338`

```javascript
if (typeof(output) !== 'string') {
     output = util.inspect(output, false, null, self.colorize);
    }

    if (meta) {
        if (typeof meta !== 'object') {
            output += ' ' + meta;
        }
        else if (meta) {
            if (this.inlineMeta) {
                output += ' ' + util.inspect(meta, false, null, self.colorize).replace(/[\n\t]\s*/gm, " ");
            }
            else {
                output += '\n' + util.inspect(meta, false, null, self.colorize);
```

**Comment by sylvainlap** [comment]
+1, got the same issue.

**Comment by troy** [comment]
This is now resolved in [v1.0.5](https://www.npmjs.com/package/winston-papertrail), which has been published to NPM. Thanks to @agoldis for the patch!
